AutomatedTesting/Gem/PythonTests/Prefab/tests/PrefabTestUtils.py

```diff
@@ -240,29 +240,71 @@ def validate_undo_redo_on_prefab_creation(prefab_instance, original_parent_id):
             "Prefab parent was not restored on Redo."
 
 
-def validate_spawned_entity_transform(entity, expected_position, expected_rotation, expected_scale):
+def validate_spawned_entity_rotation(entity, expected_rotation):
```

Comment:
These validate functions can be used for any entity and not just spawned entities. These can be moved to entity utils so that other tests can also use these.